packages/runtime/container-runtime/src/gc/gcDefinitions.ts

```diff
@@ -481,8 +463,8 @@ export interface IGCRuntimeOptions {
  */
 export interface IGarbageCollectorConfigs {
 	/**
-	 * Tracks if GC is enabled for this document. This is specified during document creation and doesn't change
-	 * throughout its lifetime.
+	 * Tracks if GC is enabled for this document. GC may not be enabled for old documents created pre-GC or for
+	 * containers that are running GC version that is older that the one in base snapshot.
```

Comment:
Are you mixing up shouldRunGC with gcEnabled?

Reply:
btw I'd like to renamed gcEnabled and sweepEnabled to gcAllowed and sweepAllowed - those are the document lifetime values, not the session-specific ones, I think allowed is a better term than enabled.

Reply:
Yeah, I mixed this with shouldRunGC.

Reply:
The renames you are suggesting seem fine. I don't want to do in this PR because that will create a lot of changes.